reactor-test/src/test/java/reactor/test/StepVerifierTests.java

```diff
@@ -1225,11 +1249,26 @@ public void longDelayAndNoTermination() {
 		            .expectNext("foo")
 		            .expectNoEvent(Duration.ofSeconds(5))
 		            .expectNextCount(1)
-		            .expectNoEvent(Duration.ofMillis(10))
-		            .thenCancel()
+		            .expectTimeout(Duration.ofHours(10))
 		            .verify();
 	}
 
+	//source: https://stackoverflow.com/questions/58486417/how-to-verify-with-stepverifier-that-provided-mono-did-not-completed
+	@Test
+	public void expectTimeoutSmokeTest() {
+		Mono<String> neverMono = Mono.never();
+		Mono<String> completingMono = Mono.empty();
+		Function<Mono<String>, Duration> f = m -> StepVerifier.create(m)
```

Comment:
nit: if we use `m -> {}`-style lambda, the `StepVerifier` block would have much shorter indentation

Reply:
I'll rewrote a bit to avoid the `Function` altogether, doesn't bring much in terms of readability